madanalysis/build/makefile_writer.py

```diff
@@ -429,6 +448,14 @@ def Makefile(
         if options.has_heptoptagger:
             file.write('LIBFLAGS += -lHEPTopTagger_for_ma5\n')
 
+        # SQLite3
+        if options.has_sqlite: 
+            file.write('LIBFLAGS += -l sqlite3\n')
```

Comment:
```suggestion
            file.write('LIBFLAGS += -lsqlite_for_ma5\n')
```
lets keep `has_sqlite` option for analysis and `has_sqlite_lib` for compiling the interface. Analysis should not be directly compiled with sqlite.